kpi-tracker/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "concordium-kpi-tracker"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.68"
+chrono = "0.4.23"
+clap = { version = "4.0.32", features = ["derive"] }
+concordium-rust-sdk = { version = "*", path = "../deps/concordium-rust-sdk", features = ["postgres"] }
+futures = "0.3.25"
+tokio = { version = "1.23.1", features = ["rt-multi-thread"] }
```

Comment:
Generally try to avoid specifying the patch version unless there is a specific bugfix that is needed in some specific version. Use `anyhow = 1.0` for example.

The lock file makes for reproducible builds.

Reply:
Makes sense. I just did `cargo add anyhow` and this is what happened. From what I can read, `1.0.68` is interpreted as `^1.0.68`. Is there a way to ignore recording the patch version when installing through the cli?

What about dependencies in the `0.X` range? Is your standpoint the same in this case?